packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -495,6 +495,9 @@ export default class New extends Component {
   }
 
   _handleSrPathSelection = async path => {
+    if (isEmpty(path)) {
```

Comment:
`_.isEmpty` checks object, collection, map, or set and in case of `No selected value`  path will be an empty string
```suggestion
    if (path === '') {
```

Reply:
`isEmpty()` also check for empty string.
lodash: `Array-like values such as arguments objects, arrays, buffers, strings, or jQuery-like collections are considered empty if they have a length of 0. Similarly, maps and sets are considered empty if they have a size of 0.`

Reply:
If that's all you need, then check directly against empty strings:

- more explicit
- code is less dependent

This does not mean we can't use that libs but it have to worth it :slightly_smiling_face: